doc/manipulating_images/data_preparation.rst

```diff
@@ -10,30 +10,53 @@ Data preparation: loading and basic signal extraction
 
 |
 
-.. topic:: **File names as arguments**
+.. currentmodule:: nilearn.image
 
-   Nilearn functions and objects accept file names as arguments::
+File names or image objects
+=============================
+
+File names and objects for 3D and 4D images
```

Comment:
> nitpick 1: perhaps something with "time series" instead of 4D as this
> is more a computerscience-vision and potentially unclear to newcomers

Fair enough: can you suggest a wording?